api/client/start.go

```diff
@@ -32,6 +32,7 @@ func (cli *DockerCli) forwardAllSignals(cid string) chan os.Signal {
 			}
 			if sig == "" {
 				fmt.Fprintf(cli.err, "Unsupported signal: %v. Discarding.\n", s)
+				continue
```

Comment:
I wonder if there was any reason behind *not* skipping `/containers/%s/kill?signal=""` call below with an unsupported signal since iirc it will default to 0 in the api handler and it will be a SIGKILL in `container.Kill()` eventually.
ping @LK4D4 @cpuguy83